app/containers/message/User.js

```diff
@@ -59,6 +64,7 @@ export default class User extends React.PureComponent {
 		}
 
 		const aliasUsername = alias ? (<Text style={styles.alias}>@{username}</Text>) : null;
+		const readReceipt = showReadReceipt ? <CustomIcon name='check' color='#0084ff' size={15} /> : <View style={styles.emptySpace} />;
```

Comment:
color: `#1d74f5`

Reply:
You should show read receipt or empty space ONLY if `Message_Read_Receipt_Enabled`.
Otherwise, you shouldn't show anything.

Reply:
@pranavpandey1998official Ok, do it.
But I'm not sure you get my point: if `read receipt` setting isn't enabled on the server, you should show anything (not even an empty space).

Reply:
Yah I completely understand that I was taking when the read-receipt setting is true and unread is also true